main.go

```diff
@@ -28,10 +31,19 @@ var (
 	gLogPath       string
 	gServerLogPath string
 	gSelect        string
-	gCommand       string
+	gCommands      arrayFlag
 	gVersion       string
 )
 
+func (a *arrayFlag) Set(v string) error {
+	*a = append(*a, v)
+	return nil
+}
+
+func (a *arrayFlag) String() string {
+	return strings.Join(*a, ", ")
```

Comment:
I don't even know where does flag package use `String` methods, but does it make more sense to join the string with `;` instead of `,` so it might also look compatible with our configuration syntax?

Reply:
Looks like the `String` method is used to determine [wether the set value is the zero value](https://golang.org/src/flag/flag.go?s=7440:7500#L447). Otherwise it is just aesthetics, I can change that no problem. The docs however state

> The flag package may call the String method with a zero-valued receiver, such as a nil pointer.

which does look dangerous here, but it doesn't seem to happen. I think it is a good idea to add a `nil`-check.

Reply:
@kmarius I think `strings.Join` simply returns an empty string with `nil` so nil-check might not be necessary. Anyway, I think we can just merge this as it it.